.github/workflows/publishOpenVSX.yml

```diff
@@ -0,0 +1,94 @@
+name: Publish in Open VSX Registry
```

Comment:
The reason why I prefer creating a seperate GHA file over adding a step in the current `publishVSCode.yml` is that:

If the publishing in the OpenVSX step failed, the whole `publishing` section will fail. However, the `ctcCloseSuccess` section needs `publishing` to be successful to move on. 

Alternatively, if we have a separate section, `publishing in Open VSX` is in the `publishVSCode.yml` file. When it fails, we can config it only to be re-run manually not to interrupt the publishing marketplace job. However, this still can't solve the `ctcCloseSuccess` dependency. When only the `publishing in Open VSX` section is run, the `ctcCloseSuccess` can't be finished because of the absence of the `publishing` section. 

Please advise if you think it can be added in the `publishVSCode.yml` file and if it's a better way to do this.